### DISCUSSION TITLE
is the mainStatus safe? because it says another client will override it, will it disable my account?

### DISCUSSION THREAD

**Discussion by nichevulcan20** [GiorgioBrux/nitro-sniper-enhanced#77] [Q&A] [answered]
while deploying on heroku, i see this The status of your main token: {online, idle, invisible, dnd}. useMain must be set to true or this is useless. This status will be overwritten by other discord clients.

will it disable my accont? because it overrides it...?

**Comment by GiorgioBrux** [comment, marked as answer]
That only means that if you set your main token to be dnd, and then (e.g.) you login with your phone, the showed status will be your phone's, not dnd.
